main.go

```diff
@@ -22,9 +27,28 @@ var (
 )
 
 func main() {
+	var dbHost string;
```

Comment:
Yes it might, but I don't need it now. The discussion how we want to pass in parameters should be had, but I'm just doing what I need to progress here.That said: config files are discouraged because we want to containerize the thing, no?